Sources/Workspace/Workspace.swift

```diff
@@ -110,80 +110,85 @@ public class Workspace {
     /// A struct representing all the current manifests (root + external) in a package graph.
     public struct DependencyManifests {
         /// The package graph root.
-        let root: PackageGraphRoot
+        internal let root: PackageGraphRoot
 
         /// The dependency manifests in the transitive closure of root manifest.
-        let dependencies: [(manifest: Manifest, dependency: ManagedDependency)]
+        internal let dependencies: [(manifest: Manifest, dependency: ManagedDependency)]
```

Comment:
Interesting approach but I don't think we should do it because of the inconsistency it brings with rest of the code base. In general, I like that we don't need to specify `internal`. Also, the fact that it doesn't leave the module, protects the APIs which are supposed to private.